doc/architecture/decisions/0003-use-nextjs-express-stylecomponents.md

```diff
@@ -0,0 +1,37 @@
+# 3. Use nextjs, express, stylecomponents
+
+Date: 2021-05-01
+
+## Status
+
+Accepted
+
+## Context
+
+There are foundational pieces required for any full-stack applications.
+For this project, we are selecting straight-forward and well known libraries
+to aid in our development.
+
+- [express.js](https://expressjs.com) is the most used `nodejs` web framework.
+- [React.js](https://reactjs.org/) is well-known and widely use `js` library
+  for building component-based user interfaces.
+- [styled-components](https://styled-components.com/) is modern `css-in-js`
```

Comment:
I might prefer [Emotion](https://emotion.sh/docs/introduction) to Styled Components — they have very similar APIs at this point, but I've found Emotion to be a bit more reliable and easier to set up. What do you think?